### PR TITLE
[NUI] Supports to build ModelNode Tree

### PR DESCRIPTION
**Pull request by bshsqa**
### Description of Change ###

This PR is to support to build ModelNode Tree after a Model finish to data from file

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI.Scene3D/src/public/Controls/Model.cs [hunk 1]
```diff
@@ -100,6 +101,14 @@ public Model(string modelUrl, string resourceDirectoryUrl = "") : this(Interop.M
         {
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
             this.PositionUsesPivotPoint = true;
+            ResourcesLoaded += (e, s) =>
```
Comment (hinohie) [inline src/Tizen.NUI.Scene3D/src/public/Controls/Model.cs:104]: Usually, we don't recommend to use instance callback in platform level.

Could you please make it as seperated method, and remove signal at `Dispose` time?

Comment (bshsqa) [reply src/Tizen.NUI.Scene3D/src/public/Controls/Model.cs]: Thank you I fixed it.

File: src/Tizen.NUI.Scene3D/src/public/ModelComponents/ModelPrimitive.cs [hunk 2]
```diff
@@ -142,8 +142,24 @@ private void SetGeometry(Geometry geometry)
         private Geometry GetGeometry()
         {
             IntPtr cPtr = Interop.ModelPrimitive.GetGeometry(SwigCPtr);
+            Geometry ret = Registry.GetManagedBaseHandleFromNativePtr(cPtr) as Geometry;
+            if (ret == null)
+            {
+                // Store the value of PositionUsesAnchorPoint from dali object (Since View object automatically change PositionUsesPivotPoint value as false, we need to keep value.)
```
Comment (hinohie) [inline src/Tizen.NUI.Scene3D/src/public/ModelComponents/ModelPrimitive.cs:148]: Looks not matched comments. remove this sentense is more clear I think.

File: src/Tizen.NUI.Scene3D/src/public/ModelComponents/ModelPrimitive.cs [hunk 3]
```diff
@@ -167,9 +183,25 @@ private void SetMaterial(Material material)
         [EditorBrowsable(EditorBrowsableState.Never)]
         private Material GetMaterial()
         {
-            IntPtr cPtr = Interop.ModelPrimitive.GetMaterial(SwigCPtr);
+            global::System.IntPtr cPtr = Interop.ModelPrimitive.GetMaterial(SwigCPtr);
+            Material ret = Registry.GetManagedBaseHandleFromNativePtr(cPtr) as Material;
+            if (ret == null)
+            {
+                // Store the value of PositionUsesAnchorPoint from dali object (Since View object automatically change PositionUsesPivotPoint value as false, we need to keep value.)
```
Comment (hinohie) [inline src/Tizen.NUI.Scene3D/src/public/ModelComponents/ModelPrimitive.cs:190]: Looks not matched comments. remove this sentense is more clear I think.

**Review by hinohie — APPROVED** [review]
Will give Approve anyway

File: src/Tizen.NUI.Scene3D/src/public/ModelComponents/ModelPrimitive.cs (hunk 2 above)
Comment (bshsqa) [reply src/Tizen.NUI.Scene3D/src/public/ModelComponents/ModelPrimitive.cs]: Thank you I removed them.

**Pull request merged**